keymanapp/keyman
# docs(ios): Add help page for Settings

Fixes #5381

This adds a help page for the Settings menu in Keyman for iPhone and iPad.

## User Testing
Setup: Install the PR build of Keyman for iPhone and iPad
* TEST_HELP_PAGE
1. Launch Keyman
2. From the menu (...), open the Info page.
3. Click 'Using Keyman for iPhone and iPad' > 'Using the Settings Screen' 
4. Verify that the 'Using the Settings Screen' page is displayed.
